### PR TITLE
used stage data as GACOS is down. add support for both ENVI and tiff.

### PR DESCRIPTION
**Pull request by dbekaert**
GACOS is down, added pre-staged data to OSL. 
Added support for both tiff and Envi

### CODE REVIEW

**Comment by dbekaert** [comment]
@sssangha Can you do a brief test of this. I tested on OSL and then moved it here for PR.

**Comment by sssangha** [comment]
@dbekaert I tested it and it looks good to me. Alert is straightforward so in the future when the website is operational, people should be able to easily leverage Tif instead of native grid

**Review by parosen — COMMENTED** [review]
After your last pull request, I accepted it, but then I made additional changes to fix typos, remove more references to master and slave, and set the kernel for unavco.  It seems the latest changes were not done before pulling first.  Is it possible to go back and make these changes on the version of Tropo.ipynb in the repository?

File: UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "hidden": true
    },
    "source": [
-    "We will start with the initial set-up of the note-book. This step only needs to be run at the very beginning and defines the processing locations as well as a few plotting routines that will be used throughout the tutorial."
```
Comment (sssangha) [inline UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb:34]: Revert this line

File: UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb [hunk 2]
```diff
@@ -249,9 +259,9 @@
     "hidden": true
    },
    "source": [
-    "The following steps require geo-coded interferogram products that are compatible with gdal and have correct geo-reference information stored in their meta-data. Gecoded **ISCE** product have a **.geo** suffix, while the addition of the **.vrt** suffix makes sure the products are also compatible with gdal. \n",
+    "The following steps require geo-coded itnerferogram products that are compatible with gdal and have correct geo-reference information stored in their meta-data. Gecoded **ISCE** product have a **.geo** suffix, while the addition of the **.vrt** suffix makes sure the products are also compatible with gdal. \n",
```
Comment (sssangha) [inline UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb:262]: Revert this line

File: UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb [hunk 3]
```diff
@@ -249,9 +259,9 @@
...
     "\n",
-    "As part of this tutorial we use a Sentinel-1 dataset which has already been pre-processed with *topsApp.py*. You can find the unwrapped interferogram (*filt_topophase.flat.geo*) and the line-of-sight angles (*los.rdr.geo*), containing incidence and heading, in the **insar/merged** data folder."
+    "As part of this tutorial we use a Sentinel-1 dataset which has already been pre-processed with *topsApp.py*. You can find the unwrapped unwrapped interferogram (*filt_topophase.flat.geo*) and the line-of-sight angles (*los.rdr.geo*), containing incidence and heading, in the **insar/merged** data folder."
```
Comment (sssangha) [inline UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb:264]: Revert

File: UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb [hunk 4]
```diff
@@ -271,7 +281,7 @@
     "hidden": true
    },
    "source": [
-    "To verify that your *FILE* is gdal compatible try to run:\n",
+    "To verify if your *FILE* is gdal compatible try to run:\n",
```
Comment (sssangha) [inline UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb:284]: revert

File: UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb [hunk 5]
```diff
@@ -956,10 +1116,10 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "unavco [conda env:.local-unavco]",
+   "display_name": "unavco",
```
Comment (sssangha) [inline UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb:1119]: revert

File: UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb [hunk 6]
```diff
@@ -956,10 +1116,10 @@
...
    "language": "python",
    "name": "conda-env-.local-unavco-py"
-   },
+  },
```
Comment (sssangha) [inline UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb:1122]: revert

File: UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb [hunk 7]
```diff
@@ -970,7 +1130,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.6"
+   "version": "3.8.10"
```
Comment (sssangha) [inline UNAVCO2021/3.3_Tropo_Ionosphere+Split_Spectrum/Troposphere/Tropo.ipynb:1133]: revert

**Review by sssangha — COMMENTED** [review]
@dbekaert since I'm unable to edit, I've flagged the lines where we need to revert back to incorporate Paul's fixes. If you prefer and give me write access, I can also take care of it.

**Review by parosen — COMMENTED** [review]
Sorry guys, I don't understand the revision interface well enough.  It seems like you are suggesting that I only reject one line of your changes, but there are a number of others.  Perhaps I just go through and reject the ones I don't think should be changed?

**Review by parosen — APPROVED** [review]

**Pull request merged**